scenarioo-client/gulpfile.js

```diff
@@ -136,8 +136,8 @@ gulp.task('test-watch', ['environmentConstants'], function (done) {
 /**
  * Run protractor ui tests (without generating Scenarioo documentation)
  */
-gulp.task('test-e2e', function () {
-    gulp.src(['./test/protractorE2E/specs/**/*.js'])
+gulp.task('e2e', function () {
+    gulp.src(['./test/e2e/specs/**/*.js'])
```

Comment:
OH THANKS GGOD!! I like that you cleaned this finally up! 👍 👍 
`protractorE2E` was a real bad directory name, very unusal.